theme/src/resources/templates/category_pages/liferay_commerce.hbs

```diff
@@ -26,11 +26,20 @@
 	$(function() {
 		const userTags = window.HelpCenter.user.tags;
 
-		const documentElementLang = document.documentElement.lang;
-
+		 var alertChildren = null;
+  
+		if ('{{help_center.locale}}' === 'ja') {
+			alertChildren = {
+					children: '{{dc 'commerce-japanese_banner_description'}}',
+					linkText: '{{dc 'commerce-japanese_banner_cta'}}',
```

Comment:
Alert only takes a leadingText parameter like on https://github.com/rdai10/zendesk/blob/76c7316390656fc6a05a7aca656fe2e236c9e4ee/theme/src/resources/templates/category_pages/liferay_dxp_7_2.hbs#L34

Reply:
We have had trouble displaying links in alerts in the past.